R/DESCRIPTION

```diff
@@ -0,0 +1,46 @@
+Package: metaflow
+Type: Package
+Title: R Binding for Metaflow 
+Version: 2.1.0
```

Comment:
Maybe just `2.0.0`? The version bumps are going to be independent between metaflow and metaflow-r.

Reply:
mark as resolved as we discussed